k8s/postgresql/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,34 +1,5 @@
 actions:
-- name: Secure login
-  bashTest:
-    script: |-
```

Comment:
Why are we removing this test?

Is secure login no longer a feature?

Reply:
In fact, that's a duplicated test. Db Operations tests are checking the connection the same way. I just updated the title.